simtong-application/src/main/kotlin/team/comit/simtong/domain/holiday/usecase/ShareHolidayUseCase.kt

```diff
@@ -34,9 +33,7 @@ class ShareHolidayUseCase(
         )
 
         val completedHolidays = holidays.map {
-            it.copy(
-                status = HolidayStatus.COMPLETED
-            )
+            it.share()
```

Comment:
> 위의 리뷰와 이어집니다.
```suggestion
            it.complete()
```
`share()` 대신 `complete()` 예시입니다.